### ISSUE TITLE
NC+

### ISSUE THREAD

**Issue by Lampardzino012** [mbebe/blomqvist#453]
witam na dwoch boxach probowalem i logowanie dziala wybieram kanal i podczas odpalania odtwarzacz pojawia sie na sekunde i sie wylacza i nie otwiera kanalow

**Comment by mbebe** [comment]
Mozesz dac jakies logi?

**Comment by Lampardzino012** [comment]
oj dalbym tylko jak to sie robi mniej wiecej

**Comment by notoco** [comment]
http://kodiwpigulce.pl/logi

**Comment by Lampardzino012** [comment]
ok jutro zobacze , a  u was nc+ dziala normalnie?

**Comment by mbebe** [comment]
Tak. Byly jakies aktualizacje i moglo cos sie porobic w boxach. RPI, Androidy cos sie zaczelo sypac. :-/

**Comment by misialek** [comment]
Na Androidzie jest wszystko 👍🏻🙂
